readme.md

```diff
@@ -1,5 +1,113 @@
+
+The Project Committee manages the day to day requirements of Projects. This includes but is not limited to the following:
+
+- Review and evaluate new Project applications
+- Monitoring health of existing Projects
+- Handling the retirement of Projects
+- Providing support to Projects.
+
+## Organizational Structure
+
+The Project Committee is comprised of volunteer .NET Foundation members and has a designated Chairperson appointed by the Executive Director. The Chairperson ensures that the Project Committee meets monthly to discuss and review items pertaining to Projects and provides information and recommendations to the Executive Director so that they can be presented to the Board of Directors for approval. All official communication from the .NET Foundation pertaining to Projects must be delivered by the Executive Director.
+
+## Evaluation Criteria
+
+Transparency is a key open source characteristic, and as a result the criteria for evaluating .NET Foundation projects is designed to be as measurable and objective as possible. The criteria is not only important for evaluating new project applications but for measuring the ongoing health of existing projects. There are two types of criteria which have been defined:
+
+**Eligibility** - this criterion is binary in nature and is used to determine if a project is fully aligned with the mission and values of the .NET Foundation.
+
+**Health** - this criterion is quantitative in nature and is based on publicly accessible metrics which can be used to evaluate the health and maturity of a project.
+
+## Project Levels
+
+Projects are categorized according to the following levels: 
+
+**Applicant** - a project which has applied to the .NET Foundation to become a member. The Project Committee will review the application within one month of the submission to determine whether or not the project has satisfied the Eligibility and/or Health requirements. Any feedback to the applicant will be delivered by the Executive Director.
+
+**Seed** - a project which has been reviewed by the Project Committee and meets the basic Eligibility requirements but does not yet meet the Health requirements. Projects at this level are eligible for some .NET Foundation project services and will be re-evaluated automatically each quarter by the Project Committee to determine if they are ready to graduate to the Member level.
+
+**Member** - a project which satisfies the Eligibility and Health requirements and has been on-boarded as an official .NET Foundation project. Projects at this level can identify themselves publicly as Member Projects and are eligible to obtain access to all .NET Foundation project services. They will continue to be monitored on a monthly basis to ensure they meet the Eligibility and Health criteria.
+
+**Retired** - a project which was once a Member Project but no longer satisfies either the Eligibility or Health requirements, or has submitted a formal request to withdraw from the .NET Foundation.
```

Comment:
This document may not be the place for it, but it could make sense for the Foundation to think through its processes and approach for when projects potentially no longer satisfy Foundation requirements. The Foundation would presumably want to communicate with projects about potential concerns before retiring a project.  Working in co-operation with projects to detail concerns where appropriate, and to support the project's continued health and membership, where possible, is presumably a sensible first goal before retiring a project.